jsk_topic_tools/src/rosparam_utils.cpp

```diff
@@ -51,6 +52,7 @@ namespace jsk_topic_tools
     }
     default:
     {
+      JSK_ROS_ERROR("the value cannot be converted into double");
```

Comment:
I think `XmlRpc::XmlRpcValue` can be printed by `<<` operator. Can you output it?